openpype/hosts/flame/api/utility_scripts/openpype_flame_to_ftrack/modules/ftrack_lib.py

```diff
@@ -138,12 +138,24 @@ def create_comonent(self, shot_entity, data, assetversion_entity=None):
 
         if name == "ftrackreview-mp4":
             duration = data["duration"]
-            handles = data["handles"]
+
+            handle_start = data.get("handleStart", None)
+            handle_end = data.get("handleEnd", None)
+            if handle_start is not None:
+                duration += handle_start
+            if handle_end is not None:
+                duration += handle_end
+            if handle_start is None and handle_end is None:
+                # Backwards compatibility; old style 'handles'
+                # We multiply by two because old-style handles defined
+                # both the handle start and handle end
+                duration += data.get("handles", 0) * 2
```

Comment:
This is not using asset document data but data from UI.
```suggestion
            handles = data["handles"]
```